scripts/gen-edpm-services-kustomize.sh

```diff
@@ -56,3 +56,8 @@ EOF
 kustomization_add_resources
 
 popd
+
+oc patch csv -n openstack-operators openstack-ansibleee-operator.v0.0.1 \
+    --type='json' -p='[{
+    "op":"replace", "path":"/spec/install/spec/deployments/0/spec/template/spec/containers/1/env/0",
+    "value": {"name": "RELATED_IMAGE_ANSIBLEEE_IMAGE_URL_DEFAULT", "value": "'${OPENSTACK_RUNNER_IMG}'"}}]'
```

Comment:
Instead of adding this here, maybe it should just be in the Makefile? Or this should be the only thing in this script, and it ought to be renamed.

Point being, creating the services kustomization and then oc applying the directory of services does not work, b/c the NodeSet controller will reconcile them back to their original value.